ESPixelStick/ESPixelStick.ino

```diff
@@ -1,22 +1,3 @@
-/*
```

Comment:
Please restore the headings 
Please restore the formatting. You removed much of the indenting. Please keep your changes to a minimum.